### DISCUSSION TITLE
Text Stamp Extension?

### DISCUSSION THREAD

**Discussion by MA-master-sec** [TurboWarp/extensions#375] [Ideas]
I hope to have a text stamp extension

Because adding character shapes to the sprite one by one is too tiring

**Comment by BlueDome77** [comment]
You could port the one from PenguinMod: https://studio.penguinmod.site/editor.html

**Reply by GarboMuffin** [reply]
The penguinmod version is a derivative of a not-open-source (although effectively source-available) extension so we cannot do that.

**Reply by BlueDome77** [reply]
Does that mean all the extensions and new blocks are unattainable?

**Reply by GarboMuffin** [reply]
I am only talking about their version of the animated text extension

**Reply by MA-master-sec** [reply]
`Animated Text`
this can help me,and i got it

**Comment by CST1229** [comment]
https://extensions.turbowarp.org/lab/text.js (you can stamp after turning a sprite into text)
